### PR TITLE
Fix CLA signatures Github action

### PR DESCRIPTION
**Pull request by luarx**
It was broken as the [last CLA change](https://github.com/safe-global/safe-eth-py/commit/61d00aefa3959a818a04ee1fc6f1da319ef0aa97) needs more modifications

### CODE REVIEW

File: .github/workflows/cla.yml [hunk 1]
```diff
@@ -21,12 +21,14 @@ jobs:
           path-to-signatures: 'signatures/version1/cla.json'
           path-to-document: 'https://safe.global/cla'
           # branch should not be protected
-          branch: 'cla-signatures'
+          branch: 'main'
```
Comment (Uxio0) [inline .github/workflows/cla.yml:24]: Why this change? We don't want the signatures to be stored in our main branch

Comment (luarx) [reply .github/workflows/cla.yml:24]: They will not be stored in the safe-eth-py repo, but in a centralised repo as we are doing with other repos that is specified in the following line ` remote-repository-name: 'cla-signatures'` (https://github.com/safe-global/cla-signatures)

**Review by Uxio0 — APPROVED** [review]
Makes sense, I didn't see the repo change! Sorry!
